xmtp_mls/src/groups/message_history.rs

```diff
@@ -78,9 +105,31 @@ impl<ApiClient> Client<ApiClient>
 where
     ApiClient: XmtpApi,
 {
+    pub fn get_sync_group(&self) -> Result<(Vec<u8>, MlsGroup), GroupError> {
+        let conn = self.store().conn()?;
+        let sync_group_id = conn
+            .find_sync_groups()?
+            .pop()
+            .ok_or(GroupError::GroupNotFound)?
+            .id;
+        let sync_group = self.group(sync_group_id.clone())?;
+
+        Ok((sync_group_id, sync_group))
```

Comment:
Do we actually need two return values here? Can't we just get the ID off of the sync group?

Reply:
very true. i'll make it just return the `MlsGroup`.